samples/serialport-arduino/global.json

```diff
@@ -0,0 +1,2 @@
+{

```

Comment:
this is so it doesn't try to pick up global.json from root which uses `2.*` sdk

Reply:
shouldn't you add here a requirement for the new SDK?

Reply:
I think I'm already mentioning to use latest sdk - I do not want to put specific version since versioning can still change. Project will produce exception when 3.0 is not found anyway

Reply:
I would probably still add the version, even if it changes in the future, that way the users will know right away if they can build the project or not, and how to fix it (since the error will tell them where to get the new sdk from). That said if you feel it's better to not have it I guess that's fine with me. I'll probably just add a comment to the file so that it doesn't get removed saying that we want to specifically not use the repo-level global.json

Reply:
ok, added